packages/models-library/src/models_library/api_schemas_webserver/catalog.py

```diff
@@ -133,3 +135,52 @@ class Config(InputSchema.Config):
 class ServiceResourcesGet(api_schemas_catalog_services.ServiceResourcesGet):
     class Config(OutputSchema.Config):
         ...
+
+
+class DEVServiceGet(ServiceGet):
+    # pylint: disable=too-many-ancestors
+
+    history: list[ServiceRelease] = Field(
+        default=[],
+        description="history of releases for this service at this point in time, starting from the newest to the oldest."
+        " It includes current release.",
+    )
+
+    class Config(OutputSchema.Config):
+        schema_extra: ClassVar[dict[str, Any]] = {
+            "example": {
+                **_EXAMPLE,  # 1.0.0
+                "history": [
+                    {
+                        "version": "1.0.5",
+                        "version_display": "Summer Release",
+                        "release_date": "2024-07-20T15:00:00",
+                    },
+                    {
+                        "version": _EXAMPLE["version"],
+                        "compatibility": {
+                            "can_update_to": "1.0.5",
+                        },
+                    },
+                    {"version": "0.9.11"},
+                    {"version": "0.9.10"},
+                    {
+                        "version": "0.9.8",
+                        "compatibility": {
+                            "can_update_to": "0.9.10",
+                        },
```

Comment:
Question: sometimes you have to update to a different service key and version. I think this is our main use case. @mguidon can confirm it. I see this will not work like that.

Reply:
For those we have to rename the keys. I already discussed those with @mguidon